lusidtools/lpt/stdargs.py

```diff
@@ -70,6 +70,13 @@ def __init__(self, description, sections=[]):
             help="choose a special LUSID environment. E.g. 'ipsum_lorem'",
         )
 
+        self.add(
+            "--apiUrl",
+            help="the relevant LUSID Api url (e.g. https://somedomain.lusid.com/api)",
```

Comment:
Do you want to say "this is only required it you are supplying a bearer token" (assume that if not, the url comes from the secrets file?)